packages/rest/src/router/swagger-route.ts

```diff
@@ -0,0 +1,44 @@
+import {RedirectRoute} from '.';
+import {RequestContext} from '../request-context';
+import {OperationArgs, OperationRetval, Request} from '../types';
+import {RestServerConfig, ApiExplorerOptions} from '../rest.server';
+import {getUrlForClient, getProtocolForRequest} from '../rest-utils';
+
+export class SwaggerRoute extends RedirectRoute {
```

Comment:
Maybe `SwaggerUIRoute` is a better name?

Reply:
+1

Reply:
not relevant after rolling back